modules/installation-azure-config-yaml.adoc

```diff
@@ -100,18 +124,25 @@ ifdef::gov[]
     region: usgovvirginia
 endif::gov[]
     resourceGroupName: existing_resource_group <9>
+    networkResourceGroupName: vnet_resource_group <10>
```

Comment:
I checked that <10> ~ <13> are repeated twice in private cluster installation doc https://61086--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_azure/installing-azure-private.html#installation-azure-config-yaml_installing-azure-private

Reply:
Lines 107-110 needs to be removed. They are getting repeated in lines 112-115.